src/components/Issues/LatestIssuesCard.scss

```diff
@@ -0,0 +1,59 @@
+.latest-issues-card {
+  &__list {
+    margin-bottom: var(--pf-v5-global--spacer--xl);
+    .pf-v5-c-list__item:not(:last-child) {
+      margin-bottom: var(--pf-v5-global--spacer--xl);
+      padding-bottom: var(--pf-v5-global--spacer--md);
+    }
+  }
+
+  &__item {
+    padding: 0;
+    margin-bottom: var(--pf-v5-global--spacer--lg);
+  }
+
+  &__icon-container {
+    flex-shrink: 0;
+    display: flex;
+    align-items: flex-start;
+  }
+
+  &__content {
+    min-width: 0;
+  }
+
+  &__icon {
+    font-size: var(--pf-v5-global--FontSize--md);
+  }
+
+  &__title {
+    font-weight: var(--pf-v5-global--FontWeight--semi-bold);
+    color: var(--pf-v5-global--Color--100);
+    line-height: var(--pf-v5-global--LineHeight--md);
+    font-size: var(--pf-v5-global--FontSize--md);
+  }
+
+  &__description {
+    color: var(--pf-v5-global--Color--200);
+    font-size: var(--pf-v5-global--FontSize--sm);
+    line-height: var(--pf-v5-global--LineHeight--md);
+    margin-bottom: var(--pf-v5-global--spacer--md);
+    margin-top: var(--pf-v5-global--spacer--sm);
+    margin-left: 0;
+  }
+
+  &__timestamp {
+    color: var(--pf-v5-global--Color--200);
+    font-size: var(--pf-v5-global--FontSize--sm);
+    line-height: var(--pf-v5-global--LineHeight--md);
+    margin-top: var(--pf-v5-global--spacer--sm);
+    margin-left: 0;
+  }
+
+  &__empty {
+    color: var(--pf-v5-global--Color--200);
+    font-style: italic;
+    text-align: center;
+    padding: var(--pf-v5-global--spacer--lg) 0;
+  }
```

Comment:
These custom styles are not needed. Please use patternfly component for layout and Text component for styling